apps/investify/pages/index.tsx

```diff
@@ -39,6 +40,7 @@ export default function Page({ title }) {
           >
             Change
           </button>
+          <UserCard />
```

Comment:
Alternatively you could render `account && <UserCard />`

Reply:
I agree that at the moment is fine to show card only if user, but I think we should rethink a default if no user connected on dashboard